pwa/src/templates/componentDetail/ComponentsDetailTemplate.module.css

```diff
@@ -20,12 +20,21 @@
   margin-block-end: var(--web-app-size-md) !important;
 }
 
-.tags {
+.layerTags {
```

Comment:
This does not contain only layerTags, maybe something along the lines of `layerAndCategoryContainer`? Working title 😄